panther_analysis_tool/main.py

```diff
@@ -48,6 +49,11 @@
 from unittest.mock import MagicMock, patch
 from uuid import uuid4
 
+from panther_analysis_tool.directory import setup_temp
+
+# this is needed at this location so each process can have its own temp directory
+setup_temp()
```

Comment:
any way this can be opt in? I can imagine scenarios where PAT is running in an environment where users dont have access to create a temp directory, and this could break potentially

Reply:
We are already using the system temp directory today in other spots, this just creates a subdirectory there per process. 

So if users had issues with temp file handling they would've seen errors prior to this change